00_Alternate_Languages/43_Hammurabi/hammurabi.bas

```diff
@@ -108,7 +108,7 @@
 900 PRINT "A FANTASTIC PERFORMANCE!!!  CHARLEMANGE, DISRAELI, AND"
 905 PRINT "JEFFERSON COMBINED COULD NOT HAVE DONE BETTER!":GOTO 990
 940 PRINT "YOUR HEAVY-HANDED PERFORMANCE SMACKS OF NERO AND IVAN IV."
-945 PRINT "THE PEOPLE (REMIANING) FIND YOU AN UNPLEASANT RULER, AND,"
+945 PRINT "THE PEOPLE (REMAINING) FIND YOU AN UNPLEASANT RULER, AND,"
```

Comment:
Typically, it is preferred to add a note to the "Porters Notes" section of the game's readme rather than editing the source code.

As this is just a simple typo, it may be fine, but I thought I'd mention this regardless